cypress/support/commands.ts

```diff
@@ -1,4 +1,5 @@
 import type { ExceptionEvent, LogEvent, MeasurementEvent, TraceEvent, TransportBody } from '@grafana/agent-core';
+import type { EventEvent } from 'packages/core/src/api/events';
```

Comment:
Shouldn't this be in the package above, exported by the core package?

Reply:
yes, thanks! wtf vscode